ChangeLog

```diff
@@ -47,6 +47,30 @@ Release date: TBA
 
   Ref #5392
 
+* ``UnsupportedAction`` has been deprecated.
```

Comment:
Should we specify that it's because of the optparse migration and what the replacement is ?

Reply:
Uhm, there is no replacement for most of these. We mention `ArgumentsProvider` and `ArgumentsManager` in another line already. So I think this is fine as is.